tests/IntegrationTest.php

```diff
@@ -98,7 +98,10 @@ public function gettingPlaintextStuffFromEncryptedGoogleShouldNotWork()
         $this->assertNotRegExp('#^HTTP/1\.0#', $response);
     }
 
-    /** @test */
+    /**
+     * @test
+     * @expectedException RuntimeException
```

Comment:
Be aware that this change has implications for the test. Now, all `RuntimeException`s are expected (eg. from the setup code) while before, only exceptions thrown from `connect()` were expected (I'm aware this change has been made for being compatible with PHPUnit 6).

Maybe test with `@expectedExceptionMessage` explicitely to avoid covering other exceptions?

Reply:
Going to add `@expectedExceptionMessage` in all changes to double check!

Reply:
You could also simply go with something like https://github.com/sebastianbergmann/phpunit/issues/2074#issuecomment-186322116 instead.

Reply:
@kelunik suggestion may have the benefit of resulting in a smaller changeset (with less potential for subtle issues as below), but this would still require some additional logic to skip this on newer PHPUnit versions.

Either way, I'm fine with either solution :+1:

Reply:
Guys, what should I do? `expectExceptionMessage` isn't available anymore on PHPUnit 6, and this assertion in variable :cry:

Reply:
How about this pseudo code? Maybe even move this to a `setExpectException()` helper method for backwards compatibility as @kelunik suggested?
```
if (phpunit >= 6) {
    $this->expectException('RuntimeException');
    $this->expectExceptionMessage("hello");
} else {
    $this->setExpectException('RuntimeException', 'hello');
}
```